docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -141,30 +141,130 @@ Property                                              Description
                                                       <https://cloud.google.com/bigquery/docs/cached-results>`_      ``false``
 ===================================================== ============================================================== ======================================================
 
-Data types
-----------
-
-With a few exceptions, all BigQuery types are mapped directly to their Trino
-counterparts. Here are all the mappings:
-
-============== =============================== =============================================================================================================
-BigQuery       Trino                           Notes
-============== =============================== =============================================================================================================
-``ARRAY``      ``ARRAY``
-``BOOLEAN``    ``BOOLEAN``
-``BYTES``      ``VARBINARY``
-``DATE``       ``DATE``
-``DATETIME``   ``TIMESTAMP(6)``
-``FLOAT``      ``DOUBLE``
-``GEOGRAPHY``  ``VARCHAR``                     In `Well-known text (WKT) <https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry>`_ format
-``INTEGER``    ``BIGINT``
-``NUMERIC``    ``DECIMAL(P,S)``                Defaults to ``38`` as precision and ``9`` as scale
-``BIGNUMERIC`` ``DECIMAL(P,S)``                Precision > 38 is not supported. Note that the default precision and scale of BIGNUMERIC is ``(77, 38)``.
-``RECORD``     ``ROW``
-``STRING``     ``VARCHAR``
-``TIME``       ``TIME(6)``
-``TIMESTAMP``  ``TIMESTAMP(6) WITH TIME ZONE`` Time zone is UTC
-============== =============================== =============================================================================================================
+.. _bigquery-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and BigQuery each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+BigQuery type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps BigQuery types to the corresponding Trino types according
+to the following table:
+
+.. list-table:: BigQuery type to Trino type mapping
+  :widths: 30, 25, 50
+  :header-rows: 1
+
+  * - BigQuery database type
+    - Trino type
+    - Notes
+  * - ``ARRAY``
+    - ``ARRAY``
+    -
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``BYTES``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DATETIME``
+    - ``TIMESTAMP(6)``
+    -
+  * - ``FLOAT``
+    - ``DOUBLE``
+    -
+  * - ``GEOGRAPHY``
+    - ``VARCHAR``
+    - In `Well-known text (WKT) <https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry>`_ format
+  * - ``INTEGER``
+    - ``BIGINT``
+    -
```

Comment:
Let's add a note about aliases. Please feel free to rephrase if necessary.
```
``INT``, ``SMALLINT``, ``INTEGER``, ``BIGINT``, ``TINYINT``, and ``BYTEINT`` are aliases for ``INT64`` in BigQuery
```

Reply:
Done